### ISSUE TITLE
Feature Request: retire not loaded buffers

### ISSUE THREAD

**Issue by nmrtv** [chrisgrieser/nvim-early-retirement#6]
### Feature Requested

Besides active and hidden, there is another buffer state - not loaded. It would be nice to close them too. You can check this state via buf.loaded property.

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by chrisgrieser** [comment]
Tbh, even after having read the docs on unloaded buffers, I do not fully understand the concept. 

This plugin should by default close all non-active buffers, i.e. buffers that are not visible in a tab or window. What case would not be covered by this?

**Comment by nmrtv** [comment]
The problem is that not loaded buffers are also invisible, but do not close, as both hidden and active properties are false. They can only be identified by the `loaded` property, which is 0. Their purpose is to create buffers that are visible in the buffer list, but their content is only loaded when they are displayed. Creating such buffers can be done with the `badd` command. But I have a specific problem with them because I use an auto-session plugin that opens buffers with not loaded status when old session is loaded.

More about this: https://vi.stackexchange.com/a/29688

The quick solution, without intrucing additional config option, is:
```lua
		local isIgnoredVisibleBuf = buf.hidden == 0 and buf.loaded == 1 and ignoreVisibleBufs
```

**Comment by chrisgrieser** [comment]
I see. well, if you already got a working solution for this, could you make a PR?

**Comment by mmirus** [comment]
Thanks, @nmrtv! I was unfamiliar with this distinction and thought the plugin just wasn't working at all. Upon closer inspection, most of my buffers after starting up nvim fall into this category, presumably also because of how my session plugin handles them. This change should help solve my case, as well.

**Comment by nmrtv** [comment]
@chrisgrieser thank you, I thought about it too. My solution was quick and dirty, but in reality the additional parameter is probably a better choice.

**Comment by nmrtv** [comment]
After quick looking at this code:
```lua
local isIgnoredUnloadedBuf = buf.loaded == 1 and ignoreUnloadedBufs
```
I think that by this logic, variable should be named `isIgnoredLoadedBuf`

**Comment by chrisgrieser** [comment]
yeah, I am not good with double negations, and the plugin settings somehow have triple negations in some cases 🙈

So I changed it according to your suggestions. Since I don't use a session plugin myself, feel free to make a PR later if you find any issues with it

**Comment by nmrtv** [comment]
@chrisgrieser because now visible is treated as active only, we need to exclude unloaded from it's condition. Add this little change and everything should work:
```lua
local isIgnoredVisibleBuf = buf.hidden == 0 and buf.loaded == 1 and ignoreVisibleBufs
```

**Comment by chrisgrieser** [comment]
done
